pages/advanced/smart-account-overview.mdx

```diff
@@ -3,8 +3,23 @@ import SafeCoreSDK from '../../assets/svg/ic-sdk.svg'
 import SafeSmartAccount from '../../assets/svg/ic-smart-account.svg'
 import CustomCard from '../../components/CustomCard'
 
-# Safe Smart Account
+## EOAs
 
+EOAs for short are Ethereum accounts that use traditional key pairs. That is, they consist of a single private key that can be used to make transactions and sign messages. If you gain access to that private key, you gain full control of the account. This means that your private key is a single point of failure that stands between you and your funds being lost.
```

Comment:
We should link to the glossary here: https://docs.safe.global/home/glossary#externally-owned-account